src/pages/ApiKeys/ApiKeysPanel.tsx

```diff
@@ -12,64 +13,56 @@ const ApiKeysPanelContainer = styled.div`
   width: 100%;
 `;
 
-class ApiKeysPanel extends Component {
-  static contextType = Context;
-  // For TS pre-3.7:
-  context!: React.ContextType<typeof Context>;
-  // For TS 3.7 and above:
-  // declare context: React.ContextType<typeof MyContext>;
-  ignoreLastFetch = false;
-
-  componentDidMount() {
-    // Fetch data initially
-    this.fetchKeys();
-  }
-
-  fetchKeys() {
-    const [state, actions] = this.context;
-    actions.fetchKeys(state.user.sessionInfo?.username ?? "", (_keys, err) => {
+const ApiKeysPanel = () => {
```

Comment:
moved back to a functional component, which is way nicer to work with, and way less code!